free-2.md

```diff
@@ -58,12 +75,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Most of the time would be spent on the language aspects since the focus of the langauge is to serve legislators when they build laws. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this is a fairly big idea since there are a lot of terms, and laws cover multiple domains that each have their own jargon and syntax.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a good idea for those who are interested in legislation. If the scope of this project was reduced to laws that pertain to a certain field, then this would probably be a good project. It might not be a good idea project because it may be simple to implement a language that can then be expanded to fit the needs of legislators.
```

Comment:
This could be a very interesting idea for a project. As you mentioned, the scope is large, so it would take a little bit (but not much) work to narrow it down. For example, you could start by finding some laws / codes that are written in 2-5 pages of English text (perhaps traffic codes?), and see if you could create a language that covers them. Alternatively, you could look at things such as the Harvey Mudd student catalog or sports rules, which are smaller than an entire set of laws. 

It's not entirely clear what would happen when a program runs (e.g., what is a conflict and how would it be identified), but I think that could be worked out over the course of the project. 

In general, though, I think this idea would be a richer project and experience than the room-scheduling project.

There is at least one other piece of related work: [The Representation Of An Evolving System Of Legal Concepts](http://ijcai.org/Past%20Proceedings/IJCAI-81-VOL%201/PDF/050.pdf)